lib/iContractWrapperBase.ts

```diff
@@ -47,6 +47,9 @@ export class ArcTransactionResult {
    * Returns null if the transaciton is not yet mined.
    */
   public async getTxMined(): Promise<TransactionReceiptTruffle | null> {
+    if (!this.tx) {
```

Comment:
why?

Reply:
Because StandardToken.approve returns an ArcTransactionResult with tx === null when the amount to approve === 0.  I've considered using this elsewhere as a means of indicating that no tx has been sent.